### PR TITLE
Phase2-hgx310B Add the possibility of cassette definition for scintillators

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add the possibility of cassette definition for scintillators

#### PR validation:

Created components of XML files from recent versions of silicon and scintillator flat files

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@srimanob Please sign this

**Comment by srimanob** [comment]
+Upgrade

Technical PR to handle cassette definitions for scintillators.

File: Geometry/HGCalCommonData/test/HGCalConvert.cpp [hunk 1]
```diff
@@ -149,11 +151,12 @@ class ConvertScintillator {
                  bool debug);
 
   const int layMin_;
+  const int cassette_;
 };
 
 int main(int argc, char* argv[]) {
-  if (argc < 6) {
-    std::cout << "Please give a minimum of 7/5 arguments \n"
+  if (argc < 7) {
```
Comment (perrotta) [inline Geometry/HGCalCommonData/test/HGCalConvert.cpp:158]: This "7" had to be propagated by hand in several parts of this code: probably better to define a constant for it

**Comment by perrotta** [comment]
+1

File: Geometry/HGCalCommonData/test/HGCalConvert.cpp (hunk 1 above)
Comment (makortel) [reply Geometry/HGCalCommonData/test/HGCalConvert.cpp:158]: Given the `return 0` in this if block, I'd say any further checks on `argc < 7` are redundant.

File: Geometry/HGCalCommonData/test/HGCalConvert.cpp [hunk 2]
```diff
@@ -207,10 +211,11 @@ int main(int argc, char* argv[]) {
     const char* outfile1 = argv[3];
     const char* outfile2 = argv[4];
     int laymin = atoi(argv[5]);
-    int debug = atoi(argv[6]);
+    int cassette = atoi(argv[6]);
+    int debug = atoi(argv[7]);
```
Comment (makortel) [inline Geometry/HGCalCommonData/test/HGCalConvert.cpp:215]: Isn't this undefined behavior if `argc == 7`?

**Pull request merged**